src/Kunstmaan/SeoBundle/Entity/Seo.php

```diff
@@ -350,7 +350,7 @@ public function getOgDescription()
      *
      * @return Seo
      */
-    public function setOgImage(Media $ogImage)
```

Comment:
Removing the type is not the fix for a null-exception. Can you change this to "Media $ogImage = null" ?